cost/flexera/cco/budget_v_actual_spend_report/README.md

```diff
@@ -21,7 +21,7 @@ This policy generates an email report comparing actual spending to budgeted valu
 This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Automation/ManagingCredentialsExternal.htm) for authenticating to datasources -- in order to apply this policy you must have a Credential registered in the system that is compatible with this policy. If there are no Credentials listed when you apply the policy, please contact your Flexera Org Admin and ask them to register a Credential that is compatible with this policy. The information below should be consulted when creating the credential(s).
 
 - [**Flexera Credential**](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm) (*provider=flexera*) which has the following roles:
-  - `billing_center_viewer`
+  - `common:org:own`
```

Comment:
This is a `permission` and not the actual `role` (i.e. org_owner).  Are you sure that users needs this high level permissio (org owner) to access the budgets API?

Reply:
In this PR I just want to show that the provided example in the error message is not correct as it doesnt let using colons in the permission.

Here is what we need as we use optima finops-analytics calls in the PT.
Can you confirm that we need a patch here :

https://github.com/flexera-public/policy_templates/blob/8a652ec60813b248cd62626e050c61b91e641ff5/.dangerfile/readme_tests.rb#L409-L428

change `/^`[a-zA-Z0-9\-_\.]+`(?:\*)?$/` to` /^`[a-zA-Z0-9\-_\.:]+`(?:\*)?$/`

Reply:
Looks like `budget_viewer` is the required role?
```suggestion
  - `budget_viewer`
```

Reply:
@bryankaraffa 
The error in linter explicitly states that `common:org:own` can be used. 
![photo_2024-05-28_14-15-06](https://github.com/flexera-public/policy_templates/assets/2770221/cf937854-2bd9-4253-8ac0-33a2fd64021a)

To be honest, I'm confused. Since linter didn't allow `optima:budget:report`, but we have `governance:applied_policy:index` in other policies:

https://github.com/flexera-public/policy_templates/blob/8a652ec60813b248cd62626e050c61b91e641ff5/automation/flexera/outdated_applied_policies/README.md#L48-L49